aiohttp_session/redis_storage.py

```diff
@@ -43,9 +43,13 @@ def save_session(self, request, response, session):
             self.save_cookie(response, key,
                              max_age=session.max_age)
         else:
-            key = str(key)
-            self.save_cookie(response, key,
-                             max_age=session.max_age)
+            if session.empty:
+                self.save_cookie(response, session._mapping,
```

Comment:
Is it ok if I push the change here for redis and open a new pr for cookie storage (so that the prs are thematically correct)?